devel/tortoisehg/Portfile

```diff
@@ -9,11 +9,11 @@ PortGroup           gitlab 1.0
 # NB: only update non-patch releases when updating Mercurial as well, as
 # TortoiseHg generally does not retain compatibility
 gitlab.instance     https://foss.heptapod.net
-gitlab.setup        mercurial/tortoisehg thg 6.6.3
+gitlab.setup        mercurial/tortoisehg thg 6.9rc0
```

Comment:
we only update to stable releases so this seems a bit premature. Also, your commit message does not follow the guidelines as in "portname: short description"

Reply:
Happy to wait for an official release and sorry I got the commit message wrong. I think things may have changed since I made my last PR.

Reply:
nothing has changed with respect to released version and commit messages for years